.github/workflows/windows_build_test.yml

```diff
@@ -49,8 +49,7 @@ jobs:
         shell:  bash -l {0}
         run: |
           conda install curl eigen
-          conda install -c conda-forge hdf5=1.10.6
-          conda remove -y yaml-cpp
```

Comment:
Should we still consider this?

Reply:
I think this change makes sense - it fixes one problem, but there is now a new linking problem on Windows because we don't find the HDF5 library at link time.  I haven't had time to figure out why.